docs/source/queries/result.md

```diff
@@ -2,62 +2,13 @@
 
 `Session::query` and `Session::execute` return a `QueryResult` with rows represented as `Option<Vec<Row>>`.
 
```

Comment:
This appears to no longer be true, as `QueryResult` has only `raw_rows` after the refactor.